index.html

```diff
@@ -1,27 +1,332 @@
+      <img class="avatar avatar-xs" src="https://picsum.photos/200/200" />
+    </section>
+
+    <section>
+      <h2>Alert</h2>
+      <p>There are 4 types of alerts. alert-error, alert-warning, alert-success and alert-info for error, warning, success
+        and info.</p>
+      <div class="alert alert-info">This is alert for info.</div>
+      <div class="alert alert-success">This is alert for success.</div>
+      <div class="alert alert-error">This is alert for an error.</div>
+      <div class="alert alert-warning">This is alert for a warning.</div>
+
+    </section>
+
+    <section>
+      <h2>Buttons</h2>
+      <p>There are 5 types of buttons. primary-btn, secondary-btn,icon-btn, link and floating-btn for primary,
+        secondary, with icon, link and floating buttons respectively.</p>
+      <div class='examples'>
+        <div>Primary button
+          <button class="btn primary-btn">click me</button>
+        </div>
+        <div>Secondary button
+          <button class="btn secondary-btn">click me</button>
+        </div>
+        <div>Button with an icon
+          <button class="btn icon-btn">
+            <span class="material-icons">home</span>
+            click me
+          </button>
+        </div>
+        <div>Link button
+          <a href="https://gooogle.com" class="link">link</a>
+        </div>
+        <div> Floating button
+          <button class="floating-btn">+</button>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <h2>Badges</h2>
+      <p>There are 4 types of badges for avatar and 1 for icons.
+        badge-lg, badge-md, badge-sm and badge-xs for large, medium, small and extra small badges respectively.</p>
+        <div class='examples'>
+        <div class="icon-with-badge">
+          <span class='material-icons'>home</span>
+          <div class="badge icon-badge">4</div>
+        </div>
+        <div class="avatar-with-badge">
+          <img class="avatar avatar-xs" src="https://picsum.photos/200/200" />
+          <div class="badge av-badge-xs"></div>
+        </div>
+        <div class="avatar-with-badge">
+          <img class="avatar avatar-sm" src="https://picsum.photos/200/200" />
+          <div class="badge av-badge-sm"></div>
+        </div>
+        <div class="avatar-with-badge">
+          <img class="avatar avatar-md" src="https://picsum.photos/200/200" />
+          <div class="badge av-badge-md"></div>
+        </div>
+        <div class="avatar-with-badge">
+          <img class="avatar avatar-lg" src="https://picsum.photos/200/200" />
+          <div class="badge av-badge-lg"></div>
+        </div>
+
+
+
+
+      </div>
+    </section>
+
+
+    <section>
+      <h2>Cards</h2>
+      <div class='examples'>
+        <div class="card text-only-card">
+          <div class="card-body">
+            <div class="text">
+              <div class="titleText">Our Changing Planet</div>
+              <div class="authorText">by Kurt Wagner</div>
```

Comment:
camel casing used for titleText and authorText could be changed to maintain uniformity

Reply:
changed to "-" notation. Thanks.